### PR TITLE
Added styling and animation

### PR DESCRIPTION
**Pull request by ShubhamGargade**

### CODE REVIEW

**Comment by ShubhamGargade** [comment]
Added submission validation and some styling

File: TokenManagementSystem/public/css/main.css [hunk 1]
```diff
@@ -82,23 +88,124 @@ h2,h1,h3{
   background-color: #9c88ff;
```
Comment (Rohan-RPJ) [inline TokenManagementSystem/public/css/main.css]: Don't do any changes here and change position from fixed to relative

File: TokenManagementSystem/public/css/teacher/createsubmissions.css [hunk 2]
```diff
@@ -8,11 +8,25 @@
...
 }
 
+
+.question_cards_item{
+  margin-right: 700px;
+  /* border: 2px solid red; */
+
+}
+
+#question1TextArea{
```
Comment (Rohan-RPJ) [inline TokenManagementSystem/public/css/teacher/createsubmissions.css]: remove questiuon1textarea

File: TokenManagementSystem/public/js/teacher/createsubmissions.js [hunk 3]
```diff
@@ -4,21 +4,20 @@ var questionCardsItem = "<li id='QNO' class='question_cards_item'>"
...
-var optionInput3 = "<input type='text' name='qQNOoption3' placeholder='Enter option 3'>";
-var optionInput4 = "<input type='text' name='qQNOoption4' placeholder='Enter option 4'>";
+var optionInput1 = "<input type='text' name='qQNOoption1' placeholder='Enter option 1' required >";
+var optionInput2 = "<input type='text' name='qQNOoption2' placeholder='Enter option 2' required >";
+var optionInput3 = "<input type='text' name='qQNOoption3' placeholder='Enter option 3' required >";
+var optionInput4 = "<input type='text' name='qQNOoption4' placeholder='Enter option 4' required >";
+var correctOptionRadio = "<input type='radio' name='qQNOcorrectOption' required >";
 var cardOptionsCloseDiv = "</div>";
-var appendButton = "<input type='button' name='append' class='btn card_btn' style='float: right;' value='Append'>";
+var appendButton = "<input type='submit' name='append' class='btn card_btn' style='float: right;' value='Append'>";
```
Comment (Rohan-RPJ) [inline TokenManagementSystem/public/js/teacher/createsubmissions.js]: change append type to button

File: TokenManagementSystem/public/js/teacher/submissions.js [hunk 4]
```diff
@@ -15,7 +15,14 @@ function getTimeValue(distance) {
       return [days, hours, minutes, seconds];
   }
 
-function getUpcomingtime() {
+function getUpcomingtime() {function events() {
```
Comment (Rohan-RPJ) [inline TokenManagementSystem/public/js/teacher/submissions.js]: remove function events here

**Pull request closed without merging**